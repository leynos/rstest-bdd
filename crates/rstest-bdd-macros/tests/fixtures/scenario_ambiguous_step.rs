use rstest_bdd_macros::{given, scenario};

#[given("a step")]
fn first() {}

#[given("a step")]
fn second() {}

#[scenario(path = "../../../../crates/rstest-bdd-macros/tests/features/ambiguous.feature")]
fn ambiguous() {}

fn main() {}
