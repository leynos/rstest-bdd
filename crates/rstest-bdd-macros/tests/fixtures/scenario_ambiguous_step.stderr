error: Ambiguous step definition for 'a step'. Matches: a step, a step
  --> tests/fixtures/scenario_ambiguous_step.rs:10:1
   |
10 | #[scenario(path = "../../../../crates/rstest-bdd-macros/tests/features/ambiguous.feature")]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `scenario` (in Nightly builds, run with -Z macro-backtrace for more info)
